source/frontend/src/features/properties/map/shared/SidebarFooter.tsx

```diff
@@ -18,18 +20,31 @@ const SidebarFooter: React.FunctionComponent<ISidebarFooterProps> = ({
   onSave,
   onCancel,
   isOkDisabled,
+  isMissingFields,
 }) => {
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const saveFields = () => {
+    onSave();
+    if (isMissingFields) {
+      setErrorMessage('Please check the required fields.');
+    }
+  };
+
   return (
     <SidebarFooterBar className="justify-content-end mt-auto no-gutters">
       {(!showEdit || editMode) && (
         <>
+          <Col xs="auto" className="pr-3" style={{ paddingTop: '0.7rem', color: 'red' }}>
+            {errorMessage}
```

Comment:
instead of storing the errorMessage in a state variable, how about conditionally rendering a section with a static error message depending on if isMissingFields is true?